Spec/Tests/RealtimeClientChannelTests.swift

```diff
@@ -3889,7 +3889,67 @@ class RealtimeClientChannelTests: XCTestCase {
             channel.attach()
         }
     }
+    
+    // RTL13b, RTB1
+    func test__131b__Channel__if_the_channel_receives_a_server_initiated_DETACHED_message_and_if_the_attempt_to_reattach_fails_then_the_channel_will_transition_to_SUSPENDED_state_with_periodic_reattach_with_channelRetryTimeout() {
+        
+        let options = AblyTests.commonAppSetup()
+        options.channelRetryTimeout = 1.0
+        options.autoConnect = false
+        let client = ARTRealtime(options: options)
+        client.internal.setTransport(TestProxyTransport.self)
+        
+        let previousRealtimeRequestTimeout = ARTDefault.realtimeRequestTimeout()
+        ARTDefault.setRealtimeRequestTimeout(1.0)
+        client.connect()
+        
+        defer { ARTDefault.setRealtimeRequestTimeout(previousRealtimeRequestTimeout); client.dispose(); client.close() }
+        
+        let transport = client.internal.transport as! TestProxyTransport
+        transport.actionsIgnored += [.attached]
 
+        expect(client.connection.state).toEventually(equal(ARTRealtimeConnectionState.connected), timeout: testTimeout)
+
+        let channel = client.channels.get(uniqueChannelName())
+        channel.attach()
+        
+        expect(channel.state).to(equal(ARTRealtimeChannelState.attaching))
+
+        let detachedMessageWithError = AblyTests.newErrorProtocolMessage()
+        detachedMessageWithError.action = .detached
+        detachedMessageWithError.channel = channel.name
+        
+        ARTUtils.seedRandom(withNumber: 5)
+        
+        var retryNumber = 1
+        let maxRetryCount = 5
+        
+        let predefinedDelays = [ 0.85, 1.16, 1.633333333333333, 1.712, 1.696 ]
```

Comment:
The meaning of these numbers isn’t clear to me – is there a way to make this clearer, by naming or commenting or both?

This applies to the similar code in `Spec/Tests/RealtimeClientConnectionTests.swift` too.

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/a929718d7bb75780a5375ba56dcf8aef8573b93a